packages/atomic/src/components/insight/index.ts

```diff
@@ -76,4 +76,6 @@ export {
   buildInteractiveResult as buildInsightInteractiveResult,
   InteractiveResult as InsightInteractiveResult,
   loadQuerySetActions as loadInsightQuerySetActions,
+  getOrganizationEndpoints as getOrganizationEndpointsInsight,
+  PlatformEnvironment as PlatformEnvironmentInsight,
```

Comment:
Mh, before we create a duplicate export, is it too late to create an 'atomic-core' bundle or something akin where functionality shared across the board lives?

Reply:
We're gonna have to do changes to bundling later anyway, so I don't feel like this should be done later. For now, I think we should leave it as-is.

Reply:
However, removing something exported will be a breaking change.

Reply:
Ah, I see what you mean.

Currently, `@coveo/atomic/insight` already exports a lot of "core" stuff, therefore I don't think we need to make an exception here. I think it'll have to be a breaking change along with everything else, when the time comes.

Reply:
Good for me! I just want to be sure that this is a conscious decision